admin/templates/base.html

```diff
@@ -209,3 +209,10 @@
 
   </body>
 </html>
+
+{% if 'test' not in request.META.HTTP_HOST and 'staging' not in request.META.HTTP_HOST and 'localhost' not in request.META.HTTP_HOST %}
```

Comment:
Hmmmmm. Concerns: 

- Supposedly request.META.HTTP_HOST went away between 1.5 and 1.6, though it clearly seems to exist in 1.11, so it's prooobably fine. 
- This is less brittle than the last implementation, but more brittle than I'd like. How about something like `contains 'admin.osf.io' and not contains '-admin.osf.io'`?
- Proxying may be a problem, but we'll find out if that's a real problem in integration testing or when it goes live.